bench/velox/optimized/BaseVector.cpp.ll

```diff
@@ -175850,14 +175850,16 @@ if.else.i:                                        ; preds = %if.then.i
   br label %_ZNK5folly3f146detail8F14TableINS1_21VectorContainerPolicyINSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEEEjvvvSt17integral_constantIbLb1EEEEE25computeChunkCountAndScaleEmbb.exit
 
 if.else11.i:                                      ; preds = %if.end
-  %sub.i = add i32 %0, -1
-  %div.i11 = udiv i32 %sub.i, 10
+  %conv = zext i32 %0 to i64
+  %sub.i = add nsw i64 %conv, -1
+  %div.i.lhs.trunc = trunc i64 %sub.i to i32
+  %div.i11 = udiv i32 %div.i.lhs.trunc, 10
```

Comment:
Regression.

Reply:
Duplicate of https://github.com/dtcxzyw/llvm-opt-benchmark/pull/600#discussion_r1597411997